CustomCraftSML/QPatch.cs

```diff
@@ -1,23 +1,24 @@
 ﻿namespace CustomCraft2SML
 {
     using System;
+    using Common;
 
     public static class QPatch
     {
         public static void Patch()
         {
-            Logger.Log("START");
+            QuickLogger.Message($"Started patching. Version {QuickLogger.GetAssemblyVersion()}");
 
             try
             {
+                //ScannerMappings.Load();
```

Comment:
Remove `ScannerMappings`. This will be handled in [SMLHelper v2.1](https://github.com/SMLHelper/SMLHelper/pull/75)